lib/v2/njust/router.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function (router) {
+    router.get('/jwc/:type?', require('./jwc'));
+    router.get('/cwc/:type?', require('./cwc'));
+    router.get('/gs/:type?', require('./gs'));
+    router.get('/eoe/:type?', require('./eoe'));
+    router.get('/dgxg/:type?', require('./dgxg'));
+    router.get('/eo/:grade?/:type?', require('./eo'));
```

Comment:
```suggestion
    router.get('/cwc/:type?', require('./cwc'));
    router.get('/dgxg/:type?', require('./dgxg'));
    router.get('/eo/:grade?/:type?', require('./eo'));
    router.get('/eoe/:type?', require('./eoe'));
    router.get('/gs/:type?', require('./gs'));
    router.get('/jwc/:type?', require('./jwc'));
```
Please sort it in alphabetical order